.circleci/unittest/linux_stable/scripts/run_test.sh

```diff
@@ -19,6 +19,11 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$lib_dir
 export MKL_THREADING_LAYER=GNU
 export CKPT_BACKEND=torch
 
+echo "TESTING GYM REGKEYS"
+python -c "import glfw"
```

Comment:
This seems to fail.
Some context:
If i'm not wrong, GLFW is a backend used by MUJOCO. We usually prefer EGL for headless rendering.
EGL can be either installed manually or we can take some docker image from cuda.
To tell MUJOCO to use EGL instead of GLFW we set MUJOCO_GL=egl and it usually does the job.